_data/collection-index.yml

```
@@ -298,3 +298,8 @@
   contact: https://github.com/trunk-io/devcontainer-feature/issues
   repository: https://github.com/trunk-io/devcontainer-feature
   ociReference: ghcr.io/trunk-io/devcontainer-feature
+- name: watchman ()
+  maintainer: vvvrrooomm
+  contact: https://github.com/vvvrrooomm/watchman/issues
+  repository: https://github.com/vvvrrooomm/watchman/
+  ociReference: ghcr.io/vvvrrooomm/watchman/watchman
```

```suggestion
  ociReference: ghcr.io/vvvrrooomm/watchman
```

This index adds an entry of the ociReference for collection metadata instead of individual Features. Thus, the author needs to only add a single entry irrespective of the number of Features that might exist in the repo. 

Can you update this? thanks!

---

Pinging on this thread as a friendly reminder! :)